trojsten/special/plugin_prask_7_1_1/mark_level_solved.py

```diff
@@ -0,0 +1,37 @@
+from trojsten.contests.models import Task
+from trojsten.submit.constants import SUBMIT_TYPE_EXTERNAL
+from trojsten.submit.models import Submit
+
+from .models import UserLevel
+
+PERMONIK_TASK_ID = 1994
+
+
+def get_task():
+    try:
+        return get_task._cache
+    except AttributeError:
+        get_task._cache = Task.objects.get(pk=PERMONIK_TASK_ID)
+        return get_task._cache
+
+
+def update_points(user):
+    solved = UserLevel.objects.filter(user=user.id, solved=True)
+    points = solved.count() * 1.5 - solved.filter(used_hint=True).count() * 0.5
```

Comment:
Mas iba `7` levelov, to znamena 7*1.5 = 10.5 bodu co nie je pekne cislo, nemozeme este nieco dplnit?

Reply:
Levelov je 10, takže to vychádza na 15b.